recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.0.2" %}
+
+package:
+    name: pyamg
+    version: {{ version }}
+
+source:
+    fn: pyamg-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyamg/pyamg-{{ version }}.tar.gz
+    md5: bd024253ebae6f7bf2d89fe32fc5d64a
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - scipy
```

Comment:
OK – I can remove ``numpy`` as a build requirement, and then it will use setuptools instead. Does that sound reasonable?